js/popover.js

```diff
@@ -65,6 +65,10 @@
     var $e = this.$element
     var o  = this.options
 
+    if (!/string|function/.test(typeof o.content)) {
+      throw new Error('Popover received a(n) ' + typeof o.content + '. Expected a string or a function.')
```

Comment:
Maybe just rephrase this to `Popover received content of type ${type}. Was expecting a string or a function.` or something like that to remove the need for "a(n)".